src/plugins/flare/flareplugin.cpp

```diff
@@ -373,10 +432,16 @@ bool FlarePlugin::write(const Tiled::Map *map, const QString &fileName)
                     out << "," << w << "," << h << "\n";
 
                     // write all properties for this object
-                    Properties::const_iterator it = o->properties().constBegin();
-                    Properties::const_iterator it_end = o->properties().constEnd();
-                    for (; it != it_end; ++it) {
-                        out << it.key() << "=" << it.value().toString() << "\n";
+                    QVariantMap propsMap = o->properties();
+                    for (QVariantMap::const_iterator it = propsMap.constBegin(); it != propsMap.constEnd(); ++it) {
+                        const char* a = it->typeName();
+                        if(strcmp(a, "Tiled::FilePath") == 0){
+                            QString z = escape(it.value());
```

Comment:
Is this working? In general, you need to use `toExportValue(it.value()).toString()` to make sure colors and file paths are handled correctly.